packit_service/worker/jobs.py

```diff
@@ -190,6 +193,25 @@ def get_config_for_handler_kls(
     return matching_jobs
 
 
+def push_initial_metrics(event: Event, handler: JobHandler, build_targets_len: int):
+    pushgateway = Pushgateway()
+
+    task_accepted_time = datetime.now()
+    response_time = (task_accepted_time - event.created_at).total_seconds()
+    pushgateway.initial_status_time.observe(response_time)
+    if response_time > 15:
+        pushgateway.no_status_after_15_s.inc()
+
+    # set the time when the accepted status was set so that we can use it later for measurements
+    event.task_accepted_time = task_accepted_time
```

Comment:
it took me some time to figure out where is the `event.task_accepted_time` being set and I find it a bit misleading it's "buried" here. I think it would be more transparent to define that value sooner, e.g. in `process_jobs()` or even during the event instance creation/parsing. What do you think?

Reply:
I implemented it here (directly after the `Task is accepted` status is set), because we discussed we want to measure the time to build in Copr from this point, when we set the statuses for the first time. So do we want to measure the copr build time from the point when the event happened? (event.created_at from Github event)

Reply:
we sorted this out over chat and the existing implementation does exactly what it's meant to do, I just got confused.